generators/action/templates/action.js.ejs

```diff
@@ -0,0 +1,4 @@
+export const <%= actionName %> = () => ({
+  type: types.<%= actionName.toUpperCase() %>,
+})
+
```

Comment:
this is for action right? if yes, ya.. must camelCase.. maybe we can include lodash in the deps to do this